src/lib.rs

```diff
@@ -5,40 +5,36 @@ extern crate conduit;
 extern crate flate2;
 
 use std::collections::HashMap;
-use std::error::Error;
 use std::io;
 use std::io::prelude::*;
 use std::path::PathBuf;
 use std::process::{Child, Command, Stdio};
 
-use conduit::{Request, Response};
+use conduit::{box_error, header, Body, HandlerResult, RequestExt, Response};
 use flate2::read::GzDecoder;
 
 pub struct Serve(pub PathBuf);
 
 impl Serve {
-    fn doit(&self, req: &mut dyn Request) -> io::Result<Response> {
+    fn doit(&self, req: &mut dyn RequestExt) -> io::Result<Response<Body>> {
         let mut cmd = Command::new("git");
         cmd.arg("http-backend");
 
         // Required environment variables
-        cmd.env(
-            "REQUEST_METHOD",
-            &format!("{:?}", req.method()).to_ascii_uppercase(),
-        );
+        cmd.env("REQUEST_METHOD", &format!("{}", req.method()));
```

Comment:
```suggestion
        cmd.env("REQUEST_METHOD", req.method().to_string());
```

I think it can be simplified.

Reply:
> I think it can be simplified.

Great catch!  I was able to get away with `as_str()` and avoid an allocation entirely.